Magento_Catalog/templates/product/price/final_price.phtml

```diff
@@ -12,33 +12,39 @@ $idSuffix = $block->getIdSuffix() ? $block->getIdSuffix() : '';
 $schema = ($block->getZone() == 'item_view') ? true : false;
 ?>
 <div class="price">
-    <?php if ($block->hasSpecialPrice()) :?>
+    <?php if ($block->hasSpecialPrice()): ?>
         <span class="price__value price__value--old">
-            <?= $block->renderAmount($priceModel->getAmount(), [
-                'display_label'     => __('Regular Price'),
-                'price_id'          => $block->getPriceId('old-price-' . $idSuffix),
-                'price_type'        => 'oldPrice',
-                'include_container' => false,
-                'skip_adjustments'  => true
-            ]); ?>
+            <?= $block->escapeHtml(
+                $block->renderAmount($priceModel->getAmount(), [
```

Comment:
please replace escapeHtml with noEscape comment for all renderAmount methods in this file